### ISSUE TITLE
[DOC] create a summary of the NARPS paper

### ISSUE THREAD

**Issue by bclenet** [Inria-Empenn/narps_open_pipelines#39]
### Goal

Create a summary of the NARPS paper.

### Why this is useful

Contributing to the NARPS Open Pipelines project requires a minimal knowledge of what was done in the NARPS study. This can be acquired by reading the [paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7771346/) as well as the [data paper](https://www.nature.com/articles/s41597-019-0113-7), although this can take time and be sometimes confusing because of the variety of topics covered by these articles.

Having a short summary of both papers, focusing on the topics that are really needed to start diving into this project, would be helpful for future contributors.

### More specifically

The summary should focus on (at least) these subjects:

- The global context / problem : analytical variability
- The idea of the paper : asking 70 teams to analyse the same dataset
- The dataset : fMRI / 108 participants / 4 runs / etc.
- The outputs : textual descriptions + team results

Of course we still want to provide links to the original papers, so that contributors have a choice.

### Open questions

Where this summary should be stored ? In the wiki ?
Which form / medium ? Text / Images / Video ?

**Comment by cmaumet** [comment]
discussed on Jan 9 with @bclenet -- we can do a documentation page (linked from the readme) that includes the bullet points above and link to more resources.
